### DISCUSSION TITLE
Trying To Create Minecraft Like Console

### DISCUSSION THREAD

**Discussion by valkyrienyanko** [spectreconsole/spectre.console#678] [Q&A] [answered]
The yellow rect is where I want to be able to write console commands like `help` `whisper` and the green rect shows where I want all the log messages in the console to appear. How hard would it be to do this?

![image](https://user-images.githubusercontent.com/6277739/147888375-f5893f91-217d-423b-8480-e8f2a6489acb.png)

This is what I have so far.
```cs
using Spectre.Console;

class Program
{
    public static void Main(string[] args)
    {
        var table = new Table();
        //table.Border = TableBorder.None;
        table.Expand();

        table.AddColumn("Log"); // This is where log messages should appear
        table.AddRow("Input"); // Where the user input is

        AnsiConsole.Write(table);

        System.Console.ReadLine();
    }
}
```

Although I'm not sure how you would expand the height of a row.

**Comment by phil-scott-78** [comment, marked as answer]
Converted this to a discussion. 

I can say that as of right now, there isn't anything built in that would accomplish this. Lots of threading concerns and it would require custom logic around the input. Doable, but it would take work.

**Reply by valkyrienyanko** [reply]
Okay thank you for the quick reply.
